### DISCUSSION TITLE
frame markers+symbols and audio

### DISCUSSION THREAD

**Discussion by goldstargloww** [keiwando/toonsquid#189] [Q&A] [answered]
i'm trying to use frame markers to spread a single audio file across multiple scenes, but they don't appear to work well with audio, any help?

(i couldnt upload this to github, sorry ^^" https://imgur.com/a/wFhpqAK )

**Comment by Cat-Butler** [comment, marked as answer]
I'm not sure frame markers are designed to work with audio, but I'll admit I haven't tried using scenes beyond visual symbol layers.

Personally, I prefer to keep my scenes as separate projects and then compositing the finished clips together with something like LumaFusion (or maybe in ToonSquid), so "well done" for the experiment!

As an alternative approach, would the [Split Audio feature](https://toonsquid.com/handbook/layers/audio/#split-audio-clip) help at all in breaking up your audio track? (Again, I haven't yet personally tried this for multiple scenes, so this is just a random thought.)

Good luck!
